lib/rhc/commands/port_forward.rb

```diff
@@ -97,9 +97,14 @@ def run(app)
             end
           end
         else
-          debug "Using #{ssh_executable} to determine forwarding ports."
-          ssh_cmd = "#{ssh_executable} #{ssh_uri.user}@#{ssh_uri.host} '#{list_ports_cmd} 2>&1'"
-          status, output = run_with_system_ssh(ssh_cmd)
+          debug "\nUsing #{ssh_executable} to determine forwarding ports."
+          ssh_cmd = "#{ssh_executable} #{ssh_uri.user}@#{ssh_uri.host} '#{list_ports_cmd}'"
+          begin
+            status, output = run_with_system_ssh(ssh_cmd)
+          rescue RHC::SSHCommandFailed
+            ex_forward_cmd = "#{ssh_executable} -N -L <source_port>:<gear_ip>:<destination_port>"
+            raise RHC::PortForwardFailedException.new("#{output + "\n" if options.debug}Error trying to collect ports to forward from app. You can try to collect the ports manually by running:\n  " + ssh_cmd + "\nThis command does not support the use of a custom ssh executable. You can try to foward the collected ports manually by running:\n  " + ex_forward_cmd)
```

Comment:
I'm seeing an odd issue here. `rhc-list-ports` outputs on STDERR, which is why we include the `2>&1` in the command. However, if there is an actual error reported from the command (such as the rhc-list-ports executable not being found in $PATH), the output ends up empty. Not sure why this happens.

I can run the commands outside of ruby and see that the output is indeed returned on STDOUT as expected. But running in rhc, this issue exists.